mod.ts

```diff
@@ -6,7 +6,7 @@ export * from "./ts/database.ts";
 export * from "./ts/result.ts";
 export { ObjectId, UpdateOptions } from "./ts/types.ts";
 export * from "./ts/util.ts";
-export const VERSION = "v0.8.0";
+export const VERSION = "v0.9.1";
```

Comment:
This needs to be VERSION = "v0.9.0"

Reply:
No - this is going to be a new version. `v0.9.0` is forever dead. When requesting `https://deno.land/x/mongo@v0.9.0/mod.ts` you will forever get the wrong version back, so that version should just be forgotten about and `v0.9.1` should be pushed out instead as a hotfix.

Reply:
### Error
```
error: Uncaught Error: downloading plugin "deno_mongo" from "https://github.com/manyuanrong/deno_mongo/releases/download/v0.9.1/deno_mongo.dll" failed.
    throw Error(`downloading plugin "${name}" from "${remoteUrl}" failed.`);
          ^
    at downloadFromRemote (https://deno.land/x/plugin_prepare@v0.7.0/mod.ts:76:11)
    at async download (https://deno.land/x/plugin_prepare@v0.7.0/mod.ts:46:7)
    at async prepare (https://deno.land/x/plugin_prepare@v0.7.0/mod.ts:60:21)
    at async init (util.ts:44:3)
    at async mod.ts:13:1
```

Reply:
Yes, because v0.9.1 has not been released yet! @manyuanrong has not merged this and v0.9.1 is not yet built. Please wait.